plugins/MauticSocialBundle/Entity/MonitoringRepository.php

```diff
@@ -26,17 +26,14 @@ public function getPublishedEntities($args = [])
         return parent::getEntities($args);
     }
 
-    /**
-     * @return float|int
-     */
-    public function getPublishedEntitiesCount()
+    public function getPublishedEntitiesCount(): int
```

Comment:
As much as I'd like to add return types, it is considered a BC break. So we would have to merge PRs that are changing public or protected method signatures in M6. At this point, we don't have anyone who would maintain the 6.x branch for a year or so. Can you revert the BC breaks so we could merge it into M5?

Reply:
Do you have a doc, where you have a list of what you consider a BC break?

Reply:
We've wrote a list here: https://contribute.mautic.org/contributing-to-mautic/developer/code/pull-requests#backward-compatibility-breaks

We should update it with more situations. Those are just what came to mind

Reply:
@escopecz  Hm, rector changes this line to the one i have in the PR. Should i revert to a previous `float|int` then?